src/app-layer-htp-file.c

```diff
@@ -248,7 +260,48 @@ int HTPFileSetRange(HtpState *s, bstr *rawvalue)
         SCLogDebug("negative end in range");
         SCReturnInt(-3);
     }
-    int retval = FileSetRange(files, crparsed.start, crparsed.end);
+
+    flags = FileFlowToFlags(s->f, STREAM_TOCLIENT);
```

Comment:
Will this always be true? I.e., direction to client?

Reply:
As far as I know, content-range is a server header only, so yes, direction will always be to client